diff --git a/bindings/python/psmoveapi.py b/bindings/python/psmoveapi.py
index 0fb1d93..f928a99 100644
--- a/bindings/python/psmoveapi.py
+++ b/bindings/python/psmoveapi.py
@@ -32,7 +32,10 @@ import sys
 import os
 import math

-library_path = os.environ.get('PSMOVEAPI_LIBRARY_PATH', None)
+library_path =  os.path.join(os.path.dirname(__file__),
+    '..', '..', 'build'
+  )
+
 if library_path:
     library_prefix = library_path + os.sep
 else:
@@ -147,11 +150,11 @@ class Controller(object):
         if name != '_controller' and hasattr(self._controller, name):
             setattr(self._controller, name, value)

-        return super().__setattr__(name, value)
+        return super(Controller, self).__setattr__(name, value)

     def __getattr__(self, name):
         if name.startswith('_'):
-            return super().__getattr__(name)
+            return super(Controller, self).__getattr__(name)
         elif name == 'serial':
             return self._controller.serial.decode('utf-8')
         elif name in ('bluetooth', 'usb'):
@@ -159,7 +162,7 @@ class Controller(object):
         elif hasattr(self._controller, name):
             return getattr(self._controller, name)

-        return super().__getattr__(name)
+        return super(Controller, self).__getattr__(name)


 class PSMoveAPI(object):
